test/Makefile.defs

```diff
@@ -1,5 +1,7 @@
 SHELL=/bin/bash -o pipefail
 
+# target IDs that support OpenMP 'requires unified_shared_memory'
+SUPPORTS_USM=gfx90a,gfx90c,gfx940,gfx941,gfx942,gfx1100,gfx1103,gfx90a:xnack+,gfx940:xnack+,gfx941:xnack+,gfx942:xnack+
```

Comment:
@carlobertolli suggested that it should not include any GPU which does not support xnack target feature. So, it should be something like:

`SUPPORTS_USM=gfx90a,gfx90c,gfx940,gfx941,gfx942,gfx90a:xnack+,gfx940:xnack+,gfx941:xnack+,gfx942:xnack+`